pkg/controller/volume/expand/BUILD

```diff
@@ -0,0 +1,54 @@
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])
```

Comment:
Add a new OWNER file for the new controller. Populate it with me, you, and Jan as approvers.